pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -24,6 +24,14 @@ def df():
     return DataFrame({"A": [0, 1], "B": [-0.61, -1.22], "C": ["ab", "cd"]})
 
 
+@pytest.fixture
+def df_ext(df):
+    df = df.copy()
+    df.loc[2, :] = [2, -2.22, "de"]
+    df = df.astype({"A": int})
+    return df
```

Comment:
What about just this?

``` python
return DataFrame({"A": [0, 1, 2], "B": [-0.61, -1.22, -2.22], "C": ["ab", "cd", "de"]})
```

I mean, there is probably no reason to couple two fixtures together.